### ISSUE TITLE
try the code

### ISSUE THREAD

**Issue by im1111** [pathak22/videoseg#3]
can you please tell me the exact steps to follow so I can be able to  execute the code because it didn't work for me

**Comment by dnaihao** [comment]
Yeah, some of the packages are out of data (e.g. imresize) and you need to modify the code a little bit to get it to work. And the code might not work properly on your local computer if you try to load a video since the numpy array result of computing nearest neighbors is too large. I added some extra code for batching frames and it seems not hurting the performance much while being able to run locally on my computer. Here is a link to the code I adapted from this repo: https://github.com/bijustin/video-segmentation/tree/master/src/uNLC
